scripts/_download.js

```diff
@@ -57,22 +72,22 @@ export async function getGitHubReleaseTagSha(org, repo, releaseTag) {
  * @returns {Promise<string>} sha256 hash of the response data
  */
 async function streamResponseToFile(targetFile, fetchResponse) {
-  const reader = Readable.fromWeb(fetchResponse.body);
+  try {
+    const reader = Readable.fromWeb(fetchResponse.body);
 
-  // setup stream to file
-  const fileStream = fs.createWriteStream(targetFile);
-  const pipeToFile = pipeline(reader, fileStream);
+    // setup stream to file
+    const fileStream = fs.createWriteStream(targetFile);
+    const pipeToFile = pipeline(reader, fileStream);
 
-  // setup stream to calculate the sha256 hash of the file
-  const hash = createHash("sha256");
-  const pipeToHash = pipeline(reader, hash);
+    // setup stream to calculate the sha256 hash of the file
+    const hash = createHash("sha256");
+    const pipeToHash = pipeline(reader, hash);
 
-  // run them both, reject if either rejects, resolve with the hash
-  try {
+    // run them both, reject if either rejects, resolve with the hash
     await Promise.all([pipeToFile, pipeToHash]);
-    return Promise.resolve(hash.digest("hex"));
-  } catch (e) {
-    return Promise.reject(e);
+    return hash.digest("hex");
+  } catch (error) {
+    throw new Error(`Error in streamResponseToFile: ${error.message}`);
```

Comment:
Why does this fn need to be wrapped in a try/catch like this?

Reply:
removed these as it belongs to https://github.com/konveyor/editor-extensions/issues/235